scene/resources/multimesh.cpp

```diff
@@ -225,6 +226,7 @@ void MultiMesh::set_instance_transform(int p_instance, const Transform &p_transf
 
 void MultiMesh::set_instance_transform_2d(int p_instance, const Transform2D &p_transform) {
 	VisualServer::get_singleton()->multimesh_instance_set_transform_2d(multimesh, p_instance, p_transform);
+	emit_signal(CoreStringNames::get_singleton()->changed);
```

Comment:
That's `emit_changed()` (from `Resource`). Then you don't need the include either.